lib/dry/web/roda/templates/Rakefile.tt

```diff
@@ -15,11 +15,30 @@ def db
   <%= config[:camel_cased_app_name] %>::Container["persistence.db"]
 end
 
+def settings
+  <%= config[:camel_cased_app_name] %>::Container["settings"]
+end
+
+def execute_database_create_command(settings)
+  database_url = settings.database_url
```

Comment:
I think is not possible, at least for the `create` task, since we need to boot the `rom` and that dependency needs already the database to be created, when trying we get `Sequel::DatabaseConnectionError: PG::ConnectionBad: FATAL`

I could use it for the `drop`